embark.el

```diff
@@ -290,11 +294,7 @@ calling convention should currently be considered unstable.
 Please keep this in mind when writing a custom indicator
 function, or when using the `which-key' indicator function from
 the wiki."
-  :type '(choice
-          (const :tag "Verbose indicator" embark-verbose-indicator)
-          (const :tag "Minimal indicator" embark-minimal-indicator)
-          (const :tag "Mixed indicator" embark-mixed-indicator)
-          (function :tag "Other")))
+  :type '(repeat function))
```

Comment:
We lost some user friendliness here.

Reply:
Yes, let's not be too friendly for now. The indicators will probably change again ;)

Reply:
OK.